plugins/link/link.cpp

```diff
@@ -31,8 +31,8 @@ struct LinkedMem {
 	wchar_t description[2048];
 };
 
-static void about(HWND h) {
```

Comment:
Isn't HWND a different size depending on architecture? Afaik only always the lower 32bit are used but still....reading https://msdn.microsoft.com/en-us/library/windows/desktop/aa383751(v=vs.85).aspx it's backed by a void* .